packages/app/src/components/catalog/EntityPage/Content/Overview.tsx

```diff
@@ -182,5 +186,11 @@ export const overviewContent = (
         </Grid>
       </EntitySwitch.Case>
     </EntitySwitch>
+
+    <EntitySwitch>
+      <EntitySwitch.Case if={isJenkinsAvailable}>
+        <EntityLatestJenkinsRunCard branch="" />
```

Comment:
```suggestion
        <Grid item xs={12}>
          <EntityLatestJenkinsRunCard branch="" />
        </Grid>
```
Wrap the Jenkins card with a grid item